### ISSUE TITLE
Intercept projectile causes reconnection error

### ISSUE THREAD

**Issue by reedom114514** [Phobos-developers/Phobos#1150]
### Description

Nothing but the title said.

### Phobos Version

B34-B36

### Conditions to reproduce

Start the multiplayer games

### INI code

```ini
[the techno]
Interceptor=yes
Interceptor.Weapon=1
Interceptor.GuardRange=6
Interceptor.VeteranGuardRange=6
Interceptor.EliteGuardRange=6
```

### Steps to reproduce

1. In a multiplayer game
2. shoot the projectile
3. Intercepting and see the RE happens

### Expected behaviour

Intercepting (as it did before B34)

### Actual behaviour

RE

### Additional context

https://www.mediafire.com/file/erstlsc452a6dlw/snapshot-20230904-220739.rar/file
https://www.mediafire.com/file/n03475uihktmnly/snapshot-20230904-220737.zip/file

### Checklist

- [X] The issue happens on the **latest official** version of Phobos and wasn't fixed yet.
- [X] I agree to elaborate the details if requested and provide thorough testing if the bugfix is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, gifs, videos, debug and exception logs, etc)*.

**Comment by Starkku** [comment]
Did the interceptor weapon have `Inviso=true` projectile or `ROT > 0` one? Were there multiple different types of interceptors?

**Comment by Starkku** [comment]
Should hopefully be fixed in b02a821, please test and report back.

**Comment by reedom114514** [comment]
> Should hopefully be fixed in [b02a821](https://github.com/Phobos-developers/Phobos/commit/b02a821a9e8a5810f1c2df88e48b1a82b02f9caf), please test and report back.

Thanks! It got fixed.
